Component/Toast/toast.css

```diff
@@ -0,0 +1,58 @@
+@import url("../../Utility CSS/navbar.css");
+
+.code-container {
+  width: 100%;
+}
+
+.btn {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  margin: 2rem;
+  padding: 4px 6px;
+  border: none;
+  border-radius: 1rem;
+  color: #fafafa;
+  background-color: none;
+}
+
+.btn-default {
+  background-color: var(--primary-color);
+  cursor: pointer;
+}
+
+.btn:active {
+  transform: scale(0.89);
+}
+
+#toast {
+  position: fixed;
+  top: 10px;
+  right: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.toast {
+  font-size: 2.5rem;
+  border-radius: 5px;

```

Comment:
could use `rem` here as well, kindly make the units consistent and prioritize using `rem`. Please update it on other places as well